internal/authz/authz.go

```diff
@@ -169,6 +191,65 @@ func (a *Authorizer) AuthorizeCreateStore(ctx context.Context) error {
 	return a.individualAuthorize(ctx, claims.ClientID, relation, System, &openfgav1.ContextualTupleKeys{})
 }
 
+// GetModulesForWriteRequest returns the modules that should be checked for the write request.
+// If we encounter a type with no attached module, we should break and return no modules so that the authz check will be against the store
```

Comment:
I don't understand this comment. If my write is 

```
folder-with-module:2, viewer, user:maria
folder:2, viewer, user:maria
```

Then because `folder` is not linked to a module, the request will be allowed? But why? I may not have permissions to write the first tuple.

Reply:
If `folder` isn't in a module, then we have to check whether the user has write access on the whole store, not just the module. If a user has access on the whole store, they have access on the individual module. @rhamzeh